tests/test_core.py

```diff
@@ -38,3 +36,100 @@ def test_get_escrow_address(self, network):
         result = kusama.get_escrow_address(buyer_addr, seller_addr)
 
         assert result == "Fgh5GQ1guNxvurv71cmHm8H5Eo8Ywrdz1mZemffAP2UrrH2"
+
+    def test_get_block(self, network):
+        kusama = Kusama()
+        kusama.connect(network=network)
+
+        block_hash = (
+            "0xa8495cdf2eaf0025966e96b06fba92f647e1e316f2abc698186ecf67919dc52b"
+        )
+
+        result = kusama.get_block(block_hash)
+
+        expected_number = 2493157
+        print(result)
+        assert result.get("block").get("header").get("number") == expected_number
+
+    def test_get_extrinsic_timepoint(self, network, mocker):
+        kusama = Kusama()
+        kusama.connect(network=network)
+
+        node_response = {
+            0: {"jsonrpc": "2.0", "result": 723219, "id": 1},
+            1: {
+                "jsonrpc": "2.0",
+                "method": "author_extrinsicUpdate",
+                "params": {
+                    "result": {
+                        "finalized": "0xa8495cdf2eaf0025966e96b06fba92f647e1e316f2abc698186ecf67919dc52b"
+                    },
+                    "subscription": 723219,
+                },
+            },
+        }
+
+        extrinsic_data = (
+            "0x35028444ac0e6cb2c7e9adfcc86919959ff044cd6f6aefcc99152592a4fe8e6d22ce7"
+            "701d86d7a5e2fdecb5a927a2e38beb91ec2d7ac97680d8271638676684c3d1f0e5fec54"
+            "215035204e2e0d6323e029cee37938c5984bdca37d1c1fa7890cdecd7a8200b0000400b"
+            "e51e7d8eb439683272967dfce076362514e8519a51164e73f21272d157b446e0700e40b5402"
+        )
+
+        get_block_return_value = {
```

Comment:
Maybe this could be moved to fixture file? With a lot of mocked responses we could end up with a big file

Reply:
No no, fixtures are supposed to have only that data that you want to be exposed to all the test functions. All data which are specific to that single function should stay in that single function.